app/assets/javascripts/dialog-holder/addon/services/dialog.js

```diff
@@ -88,6 +73,18 @@ export default class DialogService extends Service {
       class: params.class,
     });
 
+    await new Promise((resolve) => next(resolve));
```

Comment:
(I realise this is just copied from above, but...)

I *think* the 'more technically correct' thing here would be to use `schedule("afterRender", resolve)`. That way, everything will be done within the same runloop, and I think it'll still fix the problem.

Maybe try it and see if everything still works? If so then great. If not, we can stick with `next()` for now 👌

Reply:
Cool, this seems to work.